docs/samples/calling/index.html

```diff
@@ -9,495 +9,518 @@
+
+            <!-- calling / incoming -->
+            <div id="incomingsection">
+              <fieldset>
+                <legend>Incoming Call</legend>
+                <div class="u-mv">
+                  <pre id="incoming-call"> No Incoming Calls</pre>
+                  <button onclick="answer()" disabled="" id="answer" class="btn--green">Answer</button>
+                  <button onclick="endCall()" disabled="" id="end" class="btn--red">End Call</button>
+                </div>
+              </fieldset>
             </div>
-          </fieldset>
-        </div>
-
-        <!-- meetings / create -->
-        <form id="meetings-create">
-          <fieldset>
-            <legend>TODO: Dial a Number</legend>
-            <p class="note">NOTE: Supported destination types (number)</p>
-            <!-- <div class="u-mv">
-                  <button id="create-calling-client" type="submit" class="btn-code">CallingClient.createClient()</button>
-                </div> -->
-          </fieldset>
-        </form>
 
-        <!-- TODO: Once we have track changes in places and logic to update devices, below section will be added back -->
-        <!-- meeting-controls / media-devices
-        <fieldset id="control-devices">
-          <legend>Media Devices</legend>
-          <div>
-            <p class="note">NOTE: Choose your sending and receiving options for the meeting.</p>
+            <fieldset>
+              <legend>Call Controls</legend>
+              <div style="margin-bottom: 1rem;">
+                <input onclick="muteUnmute()" type="button" value="Mute" class="btn-code" id="mute_button"></input>
+                <input onclick="holdResume()" type="button" value="Hold" class="btn-code" id="hold_button"></input>
 
-            <button id="sd-get-media-devices" type="button" onclick="clearMediaDeviceList(); getMediaDevices()">Get
-              Devices</button>
+                <input id="dtmf_digit" name="dtmf_digit" value="DTMF Digit" type="string">
+                <button id="send-digit" type="button" onclick="sendDigit()" class="btn-code">Send Digit</button>
+              </div>
 
-            <div class="u-mv">
-              <label>
-                <input type="checkbox" name="ts-media-direction" value="receiveAudio" checked="">receiveAudio
-              </label>
-              <label>
-                <input type="checkbox" name="ts-media-direction" value="receiveVideo" checked="">receiveVideo
-              </label>
-              <label>
-                <input type="checkbox" name="ts-media-direction" value="sendAudio" checked="">sendAudio
-              </label>
-              <label>
-                <input type="checkbox" name="ts-media-direction" value="sendVideo" checked="">sendVideo
-              </label>
-            </div>
-          </div>
+              <!--- Transfer section -->
+              <div id="transfersection">
+                <fieldset>
+                  <legend>Call Transfer</legend>
```

Comment:
We can just keep the Call Transfer title, we can remove the outer border associated with the title and bring it to the same indentation level as other buttons

Reply:
Done